Makefile

```diff
@@ -128,7 +128,7 @@ functest: manifests generate fmt vet envtest ## Run tests.
 .PHONY: integrationtest
 integrationtest: ## Run integration tests
 	cd tests && \
-	go run github.com/onsi/ginkgo/v2/ginkgo --tags=test_integration -- -kubeconfig=${KUBECONFIGPATH} -k8sApiServerHost=${K8SAPISERVERHOST} -DSPANamespace=${DSPANAMESPACE} -DSPAPath=${DSPAPATH} -ginkgo.v
+	go test ./... --tags=test_integration -v -kubeconfig=${KUBECONFIGPATH} -k8sApiServerHost=${K8SAPISERVERHOST} -DSPANamespace=${DSPANAMESPACE} -DSPAPath=${DSPAPATH}
```

Comment:
not sure we should be using verbose mode here

Reply:
Isn't `-gingko.v` in the previous command also verbose mode though? Which is why I set this command to verbose IIRC.

Reply:
hmm so unittest has -v but functest doesn't.  we should converge on one but for now this is fine i suppose